packages/core/src/tracing/transaction.ts

```diff
@@ -66,6 +66,8 @@ export class Transaction extends SpanClass implements TransactionInterface {
     this._trimEnd = transactionContext.trimEnd;
 
     // this is because transactions are also spans, and spans have a transaction pointer
+    // TODO (v8): Replace this with another way to set the root span
```

Comment:
Is there even a todo for v8 here? Won't we get rid of the transaction class?

Reply:
Hmm I mean we somehow still need to construct a transaction for the event payload so I guess it's not gonna vanish completely. We'll just need to some way to set and find the root span/txn of a span in v8.